### ISSUE TITLE
Narrator is picking search completed for when navigating to a TextField on latest version of Edge

### ISSUE THREAD

**Issue by ssingh28** [OfficeDev/office-ui-fabric-react#5281]
MS Fabric React TextField is not working properly on SPO tenants default serach page .../_layouts/15/search.aspx on Edge with Narrator ON. It reads search completed when using keyboard someone tries to navigate to the textfield. This problem seems to be only in the latest version of the Edge Microsoft Edge 42.17134.1.0. This does not arise in the older version of Edge  41.16299.492.0.  This is happening on microsoft tenant.

**Comment by dzearing** [comment]
@ssingh28 why would this be a Fabric React bug? This sounds like a product implementation issue.

**Comment by dzearing** [comment]
An input box is not going to give you localized narrator strings. The product code should be providing the right values for ariaLabel, no? Is there a code bug in SharePoint or in the actual vanilla component?

**Comment by dzearing** [comment]
If you can please provide codepen examples for issues, that would clarify if the control has an issue, or if the product has an issue. Please evaluate the right place to push issues to and resubmit if you think there is an underlying component problem.

**Comment by ssingh28** [comment]
@dzearing : I can give you access to the page where it is occuring. Or you can repro it on your own sharepoint tenant by deploying a sharepoint extension which has a textfield rendered over a panel. And access this extension on the default search page (_layouts/15/search.aspx) of the tenant on Edge browser with Narrator ON.

**Comment by JasonGore** [comment]
When you create issues in Fabric's Github, there are numerous template links provided with core Fabric examples. They can be helpful for you in identifying behavior unique to Fabric vs. an app.

It should be far easier for you to replicate an example showing the behavior you indicate than for us to download and sync your application code.

I have reproduced the template links here with many examples to choose from:
- https://codepen.io/FabricReact/
- http://codepen.io/dzearing/pens/public/?grid_type=list
